src/legacy/core_plugins/vega/public/_vega_vis.scss

```diff
@@ -106,6 +106,10 @@
 // Adapted from https://github.com/vega/vega-tooltip
 
 .vgaVis__tooltip {
+
+  $vegaTooltipMaxWidth: $euiSize * 25;
+  max-width: $vegaTooltipMaxWidth + $euiSizeL;
```

Comment:
This is not going to fit on a mobile device of only `320px` wide. We need to be able to set a max-width of `100%` when below the desktop max-width needs.